content/es/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html

```diff
@@ -0,0 +1,37 @@
+---
+title: Tutorial interactivo - Crear un clúster
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                Para interactuar con la Terminal, use la versión de escritorio / tableta
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1" data-katacoda-lang="es" data-katacoda-id="kubernetes-bootcamp/1" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-font="Roboto" data-katacoda-fontheader="Roboto Slab" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;"></div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/es/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">Continuar con Modulo 2<span class="btn__next">›</span></a>
```

Comment:
```suggestion
                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">Continuar con el Módulo 2<span class="btn__next">›</span></a>
```

The Spanish page for `docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/` doesn't exist yet, the suggestions changes the URL to the English version.

The other change is `Modulo`, that is accentuated M**ó**dulo in Spanish.